### ISSUE TITLE
Make sample app compile under Xcode 4

### ISSUE THREAD

**Issue by JaviSoto** [mindsnacks/MSActiveConfig#21]
I created the app with Xcode 5 and the nib files have a different format that Xcode 4 can't open.

**Comment by aaronbrethorst** [comment]
:thumbsup: I can't publish this on Cocoa Controls without a screenshot and can't get a screenshot right now without Xcode 4 support :)

**Comment by JaviSoto** [comment]
The problem is that I would need to re-do the UI on Xcode 4 :) When I did it I didn't know the new .xib format wasn't retrocompatible. Do you want me to take a screenshot using Xcode 5?

**Comment by aaronbrethorst** [comment]
Yeah, that would work too :) No chrome around the screenshot; just use Command+S in the Simulator.

thanks!

**Comment by JaviSoto** [comment]
![ios simulator screen shot jul 30 2013 6 16 48 pm](https://f.cloud.github.com/assets/666807/883280/e93fd07c-f97e-11e2-937a-7ddcb261db57.png)
